ReactWindows/ReactNative/Animated/FrameBasedAnimationDriver.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System;
+
+namespace ReactNative.Animated
+{
+    class FrameBasedAnimationDriver : AnimationDriver
+    {
+        private readonly double[] _frames;
+        private readonly double _toValue;
+
+        private long _startFrameTimeTicks = -1;
+        private double _fromValue;
+
+        public FrameBasedAnimationDriver(int id, ValueAnimatedNode animatedValue, ICallback endCallback, JObject config)
+            : base(id, animatedValue, endCallback)
+        {
+            _frames = config.GetValue("frames").ToObject<double[]>();
```

Comment:
Generally not worried about it, It's not user-generated JSON, the data validation should occur JS side in most cases.

---
In reply to: [74443274](https://github.com/ReactWindows/react-native-windows/pull/583#discussion_r74443274) [](ancestors = 74443274)